external/dhcpc/dhcpc.c

```diff
@@ -470,6 +481,7 @@ static void dhcpc_close(void *handle)
 
 static struct dhcpc_state *g_pResult;
 int g_dhcpc_state;
+uint8_t itr;
```

Comment:
Multiple declarations for loop iterator,
322(local to dhcpc_parseoptions()) and 484 (global, used only in dhcpc_request())
Better to declare iterators inside the function only at the very beginning.